modules/alloy/src/main/ts/ephox/alloy/positioning/view/PositionCss.ts

```diff
@@ -26,6 +26,19 @@ const NuPositionCss = (
 const applyPositionCss = (element: SugarElement, position: PositionCss): void => {
   const addPx = (num: number) => num + 'px';
 
+  /*
```

Comment:
If you want to animate the context toolbar between a state of top and bottom, I think you'll need to create an intermediate state that matches the flow of the destination, and reflow before it. I didn't have time to investigate exactly what you'd need to do this. Origin.reposition might have some approaches.

Reply:
Another point on transitions

The context toolbar seems to lag behind when the outer page scrolls, but not when the editor scrolls. I suspect this is to do with scroll on the outer page being throttled, but the editor scroll not being throttled. Haven't verified this, however.